### PR TITLE
Feature/conditional compilation ci full

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: config-overrides/manifest.overrides.js [hunk 1]
```diff
@@ -24,7 +18,7 @@ function firefoxGeckoview(manifest) {
 /**
  * @param {typeof base} manifest
  */
-function firefoxDesktop(manifest) {
+function firefoxDesktopAndAndroid(manifest) {
```
Comment (Artoria2e5) [inline config-overrides/manifest.overrides.js]: Just remove this one and call it "firefox"

**Comment by Artoria2e5** [comment]
然后我不知道你有没有发现，我们 cond compile 这个还没 merge 进去就开子 master branch 说明什么？说明这几个 branch 开早了。本来是可以 `checkout -b` 的事情。

File: config-overrides/manifest.overrides.js (hunk 1 above)
Comment (Jack-Works) [reply config-overrides/manifest.overrides.js]: it's confusing because we have gecko which also is firefox

Comment (Artoria2e5) [reply config-overrides/manifest.overrides.js]: it's not hard, it's just a `geckoViewManifest extends firefoxManifest` concept

File: scripts/ci-build.js [hunk 2]
```diff
@@ -52,20 +52,18 @@ function getBuildCommand(platforms) {
         if (type === 'chromium' && platforms.indexOf('base') !== -1) {
             // chromium doesn't have it's own changes yet.
             // just copying base version is acceptable
-            return `
-            cp Maskbook.base.zip Maskbook.chromium.zip
-            `
+            return ''
```
Comment (Artoria2e5) [inline scripts/ci-build.js]: ……………………………………

**Comment by Jack-Works** [comment]
> Just let it be, as we are already using them in the mobile branch.

Hmm Should I close this pr or ...?

**Comment by Jack-Works** [comment]
> 然后那个 `return ''` 是搞什么鬼……我是让你写成一行的字符串不是让你搞成真的空字符串

It has the same effect when it applies to chrome

**Pull request merged**